wiki/Universal_offset/id.md

```diff
@@ -0,0 +1,13 @@
+---
+stub: true
+---
+
+# Universal offset
+
+*Untuk penggunaan lainnya, kunjungi [Offset (disambiguasi)](/wiki/Disambiguation/Offset).*
+
+*Untuk petunjuk langkah demi langkah tentang cara mengatur nilai Universal offset yang benar, lihat [Bagaimana cara menggunakan Offset Wizard](/wiki/Guides/How_to_Use_the_Offset_Wizard).*
+
+**Universal offset** adalah [pengaturan](/wiki/Options) klien osu! yang mempengaruhi timing pada keseluruhan [beatmap](/wiki/Beatmap) di dalam klien osu!. Menyetelnya ke nilai selain `0`, akan menambahkan jeda antara setiap lagu pada sebuah beatmap dan suara grafik yang terkait lainnya. Menyetel Universal offset ke nilai positif akan membuat suara dan grafik yang terkait dengan beatmap muncul lebih awal, sementara menyetelnya ke nilai negatif akan membuatnya muncul kemudian.
```

Comment:
```suggestion
**Universal offset** adalah [pengaturan](/wiki/Options) offset audio yang mempengaruhi timing pada keseluruhan [beatmap](/wiki/Beatmap) di dalam klien osu!. Menyetelnya ke nilai selain `0`, akan menambahkan jeda antara setiap lagu pada sebuah beatmap dan suara grafik yang terkait lainnya. Menyetel Universal offset ke nilai positif akan membuat suara dan grafik yang terkait dengan beatmap muncul lebih awal, sementara menyetelnya ke nilai negatif akan membuatnya muncul kemudian.
```

Reply:
i don't know if this is intentional or not, but if so, can you provide the reasons of why is that?